src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -298,7 +300,8 @@ function getTerminalServices(accessor: ServicesAccessor): ITerminalServicesColle
 		instanceService: accessor.get(ITerminalInstanceService),
 		editorService: accessor.get(ITerminalEditorService),
 		profileService: accessor.get(ITerminalProfileService),
-		profileResolverService: accessor.get(ITerminalProfileResolverService)
+		profileResolverService: accessor.get(ITerminalProfileResolverService),
+		accessibleViewService: accessor.get(IAccessibleViewService)
```

Comment:
I was only keeping the terminal services here, for the accessibility one we can pull it off the accessor in each action like this:

https://github.com/microsoft/vscode/blob/6fb7d0ae591d1076e1049c89d2187ab33558dab4/src/vs/workbench/contrib/terminal/browser/terminalActions.ts#L1505-L1506

Reply:
The same ID thing is a little concerning, I'm guessing it would cause a regression somewhere. Can we just add a separate action and duplicate the keybindings instead? The real fix here is a MultiCommand if you want to try that route (which I'm not too familiar with).